docs/_posts/kolia1985/2021-12-09-text_detection_v1_en.md

```diff
@@ -25,23 +25,91 @@ CRAFT: Character-Region Awareness For Text detection, is designed with a convolu
 
 {:.btn-box}
 <button class="button button-orange" disabled>Live Demo</button>
-<button class="button button-orange" disabled>Open in Colab</button>
+[Open in Colab](https://colab.research.google.com/github/JohnSnowLabs/spark-ocr-workshop/blob/master/tutorials/Certification_Trainings/others/SparkOcrImageTextDetection.ipynb){:.button.button-orange.button-orange-trans.co.button-icon}
```

Comment:
Colab notebook link is broken. Could you please check?

Reply:
solved